WordPress/gutenberg
# Enhancement: Post Terms Without Links

## What?

- Added a new block attribute called "isLink" (boolean) 
- Added a new block attribute called "linkTarget" (string) 
- Removes the terms links on block SSR and wraps the names of the terms in "span" tags (for styling purposes and also differentiate from separator) when the "isLink" attribute is false

Related to #50904

## Why?

- When using terms that should not link to archives, there is no option to remove the links from the terms output.

## How?

- A new optional attribute

## Testing Instructions




- Add categories to a post.
- Insert the "Post Category" block in the post content.
- Use the new toggle that appears in the "Advanced" panel from the sidebar.

### Testing Instructions for Keyboard


## Screenshots or screencast 
![gutenberg-terms-links](https://github.com/WordPress/gutenberg/assets/7514504/91e07c3b-ac0d-4f95-b7d4-cdd891eca3f0)
